### PR TITLE
Including documentation for PySpark Support

### PR DESCRIPTION
**Pull request by ifilonenko**
Documentation for PySpark Support
@erikerlandson @foxish

### CODE REVIEW

**Comment by erikerlandson** [comment]
LGTM, thanks!

File: src/jekyll/running-on-kubernetes.md [hunk 1]
```diff
@@ -159,6 +159,73 @@ remote URIs. Also, application dependencies can be pre-mounted into custom-built
...
+    CMD SPARK_CLASSPATH="${SPARK_HOME}/jars/*" && \
+        if ! [ -z ${SPARK_MOUNTED_CLASSPATH+x} ]; then SPARK_CLASSPATH="$SPARK_MOUNTED_CLASSPATH:$SPARK_CLASSPATH"; fi && \
+        if ! [ -z ${SPARK_SUBMIT_EXTRA_CLASSPATH+x} ]; then SPARK_CLASSPATH="$SPARK_SUBMIT_EXTRA_CLASSPATH:$SPARK_CLASSPATH"; fi && \
+        if ! [ -z ${SPARK_EXTRA_CLASSPATH+x} ]; then SPARK_CLASSPATH="$SPARK_EXTRA_CLASSPATH:$SPARK_CLASSPATH"; fi && \
+        if ! [ -z ${SPARK_MOUNTED_FILES_DIR} ]; then cp -R "$SPARK_MOUNTED_FILES_DIR/." .; fi && \
+        exec /sbin/tini -- ${JAVA_HOME}/bin/java $SPARK_DRIVER_JAVA_OPTS -cp $SPARK_CLASSPATH \
+        -Xms$SPARK_DRIVER_MEMORY -Xmx$SPARK_DRIVER_MEMORY \
+        $SPARK_DRIVER_CLASS $PYSPARK_PRIMARY MY_PYSPARK_FILE,$PYSPARK_FILES $SPARK_DRIVER_ARGS
+      
 ### Accessing Kubernetes Clusters
```
Comment (liyinan926) [inline src/jekyll/running-on-kubernetes.md]: Ditto. This does not fall under dependency management so should be `##`.

File: src/jekyll/running-on-kubernetes.md [hunk 2]
```diff
@@ -159,6 +159,73 @@ remote URIs. Also, application dependencies can be pre-mounted into custom-built
 can be added to the classpath by referencing them with `local://` URIs and/or setting the `SPARK_EXTRA_CLASSPATH`
 environment variable in your Dockerfiles.
 
+### Python Support
```
Comment (liyinan926) [inline src/jekyll/running-on-kubernetes.md]: Is this supposed to be at the same level as `Dependency Management Without The Resource Staging Server`?

File: src/jekyll/running-on-kubernetes.md (hunk 1 above)
Comment (ifilonenko) [reply src/jekyll/running-on-kubernetes.md]: Good call. I will modify accordingly

**Comment by liyinan926** [comment]
LGTM.

**Pull request merged**